docs/standard/microservices-architecture/secure-net-microservices-web-applications/authorization-net-microservices-web-applications.md

```diff
@@ -63,15 +63,15 @@ public ActionResult API1 ()
 
 In this example, to call API1, a user must:
 
--   Be in the Adminstrator *or* PowerUser role, *and*
+- Be in the Administrator *or* PowerUser role, *and*
 
--   Be in the RemoteEmployee role, *and*
+- Be in the RemoteEmployee role, *and*
 
--   Satisfy a custom handler for CustomPolicy authorization.
+- Satisfy a custom handler for CustomPolicy authorization.
 
-## Implementing policy-based authorization
+## Implement policy-based authorization
 
-Custom authorization rules can also be written using [authorization policies](https://docs.asp.net/en/latest/security/authorization/policies.html). In this section we provide an overview. More detail is available in the online [ASP.NET Authorization Workshop](https://github.com/blowdart/AspNetAuthorizationWorkshop).
+Custom authorization rules can also be written using [authorization policies](https://docs.asp.net/en/latest/security/authorization/policies.html). In this section, we provide an overview. More detail is available in the online [ASP.NET Authorization Workshop](https://github.com/blowdart/AspNetAuthorizationWorkshop).
```

Comment:
The additional comma wasn't necessary.

Reply:
Link here should be updated to /aspnet/core/security/authorization/policies

In this section, we provide an overview. -> This section provides an overview.

More detail is available in the online [ASP.NET Authorization Workshop] (https://github.com/blowdart/AspNetAuthorizationWorkshop). -> For more information, see [ASP.NET Authorization Workshop] (https://github.com/blowdart/AspNetAuthorizationWorkshop).

Reply:
Fixed.